From 863b1d54c0c4cae66e5448cdab3a96041a84d106 Mon Sep 17 00:00:00 2001
From: Lennart Poettering <lennart@poettering.net>
Date: Wed, 2 May 2018 16:54:32 +0200
Subject: [PATCH] man: document the XDG specs as further sources of
 specifications for file-hierarchy(7)

We document this further down in the text, but let's also list this
early on, where we mention the FHS as major influence too, so that it is
clear we incorporate all that thinking.

(cherry picked from commit 03f2b38e0c44343277f90c5fe761db5696ce52ba)
---
 man/file-hierarchy.xml | 20 ++++++++++----------
 1 file changed, 10 insertions(+), 10 deletions(-)

diff --git a/man/file-hierarchy.xml b/man/file-hierarchy.xml
index 3e28fc6bee..07cd458f70 100644
--- a/man/file-hierarchy.xml
+++ b/man/file-hierarchy.xml
@@ -53,16 +53,16 @@
     <title>Description</title>
 
     <para>Operating systems using the
-    <citerefentry><refentrytitle>systemd</refentrytitle><manvolnum>1</manvolnum></citerefentry>
-    system and service manager are organized based on a file system
-    hierarchy inspired by UNIX, more specifically the hierarchy
-    described in the <ulink
-    url="http://refspecs.linuxfoundation.org/FHS_3.0/fhs-3.0.html">File
-    System Hierarchy</ulink> specification and
-    <citerefentry project='man-pages'><refentrytitle>hier</refentrytitle><manvolnum>7</manvolnum></citerefentry>.
-    This manual page describes a more minimal, modernized subset of
-    these specifications that defines more strictly the suggestions
-    and restrictions systemd makes on the file system
+    <citerefentry><refentrytitle>systemd</refentrytitle><manvolnum>1</manvolnum></citerefentry> system and service
+    manager are organized based on a file system hierarchy inspired by UNIX, more specifically the hierarchy described
+    in the <ulink url="http://refspecs.linuxfoundation.org/FHS_3.0/fhs-3.0.html">File System Hierarchy</ulink>
+    specification and <citerefentry
+    project='man-pages'><refentrytitle>hier</refentrytitle><manvolnum>7</manvolnum></citerefentry>, with various
+    extensions, partially documented in the <ulink
+    url="http://standards.freedesktop.org/basedir-spec/basedir-spec-latest.html">XDG Base Directory
+    Specification</ulink> and <ulink url="https://www.freedesktop.org/wiki/Software/xdg-user-dirs/">XDG User
+    Directories</ulink>. This manual page describes a more generalized, though minimal and modernized subset of these
+    specifications that defines more strictly the suggestions and restrictions systemd makes on the file system
     hierarchy.</para>
 
     <para>Many of the paths described here can be queried
-- 
2.17.0

